recipes/squashfs-tools-feedstock/recipe/meta.yaml

```diff
@@ -11,6 +11,7 @@ package:
 source:
   # url: http://deb.debian.org/debian/pool/main/s/squashfs-tools/squashfs-tools_4.3.orig.tar.gz
   {% if variant == '_linux' %}
+  {% set best_comp = 'zstd' %}
```

Comment:
Is there a way for us to just set the default compression algo to be the same for all the platforms?  Feels like preferring zstd for performance vs xz would be preferable?

Reply:
This is just wip stuff.. I intend finally to use the upstream source and not the Mac port.. just need to make it all work first..

Reply:
You can set VARIANT env var to linux to use upstream source or anything else to use the Mac port, even on Linux atm.

Reply:
Not anymore, I've gone with squashfskit for now!

Reply:
And I've gone back to squashfs-tools due to the credit debacle!